nservicebus/rabbitmq/configuration-api.md

```diff
@@ -107,4 +107,25 @@ If the routing topologies mentioned above isn't flexible enough then take full c
  1. Define the topology by creating a class implementing `IRoutingTopology`
  1. Register it with the transport calling `.UseRoutingTopology` as shown below
 
-snippet:rabbitmq-config-useroutingtopology
\ No newline at end of file
+snippet:rabbitmq-config-useroutingtopology
+
+## Transactions and delivery guarantees
+
+### Version 6 and above
+
+The RabbitMQ transport supports the following [Transport Transaction Modes](/nservicebus/messaging/transactions.md):
+
+* Receive Only
+* No transactions
+
+### Receive Only
+
+When running in Receive Only mode, the RabbitMQ transport consumes messages from the broker in manual acknowledgment mode. After a message is successfully processed, it is acknowledged via the AMQP [basic.ack](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.ack) method, which lets the broker know that the message can be removed from the queue. If a message is not successfully processed and needs to be retried, it is requeued via the AMQP [basic.reject](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.reject) method.
+
+If the connection to the broker is lost for any reason before a message can be acknowledged, the message will automatically be requeued by the broker, letting it be retried by the endpoint.
+
+### No transactions
+
+When running without transactions, the RabbitMQ transport consumes messages from the broker in manual acknowledgment mode. Regardless of whether a message is successfully processed or not, it is acknowledged via the AMQP [basic.ack](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.ack) method after the processing attempt. This means that a message will be attempted once, and moved to the error queue if it fails.
```

Comment:
> In this mode, the acknowledgement to the broker is done right away even before the message is processed by the endpoint's message handler. In this scenario, if the processing of the message fails, then the message will be forwarded to the error queue.

That's not how it works though. It really is identical. The only difference is that since FLR is disabled in 'None` the you always ack the message.